test/test_real_pebble.py

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/python3
+# Copyright 2021 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+import shutil
+import tempfile
+import threading
+import time
+import unittest
+import urllib.error
+import urllib.request
+import uuid
+
+from ops import pebble
+
+from .test_testing import _PebbleStorageAPIsTestMixin
```

Comment:
@benhoyt how do you feel about this? Would you rather the mixin lost the leading underscore? Are you ok with cross-module importing in the tests, rather than moving it into a separate module that both the test modules import from?

Reply:
I think the import is fine, but yeah, let's drop the `_`.